CHANGELOG.md

```diff
@@ -1,3 +1,40 @@
+CHANGES IN VERSION 1.11.0 (from 1.10.0)
+===================================
+
+Appium 1.11.0 is a minor release
+
+#### General
+
+#### iOS
+
+#### Android
+* Add `chromedriverDisableBuildCheck` capability [#474](https://github.com/appium/appium-android-driver/pull/474)
+* Add `skipDeviceInitialization` capability [#480](https://github.com/appium/appium-android-driver/pull/480)
+* Fall back to pressing BACK button if ESC fails to close soft keyboard [#471](https://github.com/appium/appium-android-driver/pull/471)
+* Fix: Include API level 23 for granting of permissions to apps. 23 was being incorrectly excluded previously [#473](https://github.com/appium/appium-android-driver/pull/473)
+* Fix: Only call `helpers.ensureDeviceLocale` if language or locale was set [#477](https://github.com/appium/appium-android-driver/pull/477)
```

Comment:
for issues like this i like to try and make clear why this is relevant to and end user. What did this fix do for them? Did it speed things up? If so, say: `Fix: ensure helpers.ensureDeviceLocale is only called if language or locale is set, to avoid spending startup time on locale unnecessarily`

Users don't really care about whether or not particular functions are called in our code.